query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java

```diff
@@ -93,7 +98,7 @@ public static void setup() throws Exception {
     kafkaZk.start();
 
     pinotServiceManager =
-        new GenericContainer<>(DockerImageName.parse("hypertrace/pinot-servicemanager:main"))
+        new GenericContainer<>(DockerImageName.parse("traceableai-docker.jfrog.io/hypertrace/pinot-servicemanager:test"))
```

Comment:
Once https://github.com/hypertrace/pinot/pull/106/files is merged, can be reverted.